build_scripts/windows/Product.wxs

```diff
@@ -61,6 +51,12 @@
     <UIRef Id="WixUI_Minimal"/>
     <UIRef Id="WixUI_ErrorProgressText"/>
 
+    <!-- Show message to restart any terminals only if the PATH is changed -->
+    <CustomAction Id="Set_WIXUI_EXITDIALOGOPTIONALTEXT" Property="WIXUI_EXITDIALOGOPTIONALTEXT" Value="Please close and reopen any active terminal window to use Azure CLI." />
+    <InstallUISequence>
+      <Custom Action="Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="CostFinalize">NOT Installed AND NOT WIX_UPGRADE_DETECTED</Custom>
+    </InstallUISequence>
```

Comment:
I saw this document: https://wixtoolset.org//documentation/manual/v3/wixui/wixui_customizations.html, and it uses

```xml
  <CustomAction Id="CA_Set_WIXUI_EXITDIALOGOPTIONALTEXT" Property="WIXUI_EXITDIALOGOPTIONALTEXT" Value="Thank you for installing [ProductName]."/>
  <InstallUISequence>
    <Custom Action="CA_Set_WIXUI_EXITDIALOGOPTIONALTEXT" After="FindRelatedProducts">NOT Installed</Custom>
  </InstallUISequence>
```
Just wondering what the difference between `CostFinalize` and `FindRelatedProducts` is.

Reply:
`CostFinalize` computers the final feature and component states. `FindRelatedProducts` should be scheduled before that, but feature and component states haven't been fully computer yet. `CostFinalize` is more typical.

It doesn't matter, though: both of these actions happen *way* before the end when the property actually gets used.

Speaking of `FindRelatedProducts`, you don't even need it and my other PR removes it. You have 1 feature. It's pointless and time-consuming to migrate feature states for 1 feature. If 1 feature is installed, the product is installed. If 0 features are installed, the product gets uninstalled. So you're not migrating anything; however, that action to actually process the old products' feature states and reapply them to the new product takes several seconds on my sandbox environment. That's time wasted.